README.md

```diff
@@ -29,16 +29,13 @@ gopay:
 	testMode       : false
 ```
 
-A přístup v presenteru pak bude díky autowiringu vypadat:
+A přístup v presenteru pak bude díky autowiringu a `@inject` anotaci vypadat:
 
 ```php
-/** @var Markette\Gopay\Service */
-private $gopay;
+use Markette\Gopay\Service;
```

Comment:
Personally I wouldn't import the whole path, because having only `Service` in code later is quite unreadable :). Importing `Markette\Gopay` is nice compromise I think. `Gopay\Service` is still clear.

Of course the deepest reason is `Service` as name of class :smiley:.